assets/large_language_models/dbcopilot/components/src/prompt_flow_creation.py

```diff
@@ -104,7 +104,8 @@ def create(
             flow_string = flow_string.replace("@@GROUNDING_EMBEDDING_URI@@", grounding_embedding_uri)
             flow_string = flow_string.replace("@@DB_CONTEXT_URI@@", db_context_uri)
             flow_string = flow_string.replace("@@DATASTORE_URI@@", datastore_uri)
-            flow_string = flow_string.replace("@@EXAMPLE_EMBEDDING_URI@@", example_embedding_uri if example_embedding_uri else "")
+            flow_string = flow_string.replace("@@EXAMPLE_EMBEDDING_URI@@",
```

Comment:
Do we need this change?

Reply:
In this file, code quality gate was failing due to line length greater than 119.
Hence adding the small fix in the same PR though it's not related to this in anyway (just to expediate process).